torch/csrc/CudaIPCTypes.h

```diff
@@ -0,0 +1,61 @@
+#pragma once
+#include <c10/core/Allocator.h>
+
+#define CUDA_IPC_REF_COUNTER_FILE_SIZE 10000
+#define CUDA_IPC_WARN_AFTER_X_BLOCKS_IN_LIMBO 1000
```

Comment:
We live in C++ now, so there's no reason to define these constants as macros. We can use `constexpr int CUDA_IPC_REF_COUNTER_FILE_SIZE = 10000;` instead

Reply:
(and putting it in the namespace means you don't have to worry about symbol conflicts.)